src/AddIns/Uno.UI.MediaPlayer.WebAssembly/HtmlMediaPlayer.cs

```diff
@@ -98,7 +98,7 @@ public bool IsAudio
 
 	public bool IsVideo
 	{
-		get => videoTagAllowedFormats.Contains(Path.GetExtension(Source), StringComparer.OrdinalIgnoreCase);
```

Comment:
`videoTagAllowedFormats` is now unused. It can be deleted.

Reply:
i remove the isVideo and the videoTagAllowedFormats.